### PR TITLE
Add clarafication comment to what is backed up

### PR DESCRIPTION
**Pull request by ml4**
This clarifies what is exactly backed up by the new backup API.

### CODE REVIEW

**Comment by nfagerlund** [comment]
Hmm, that's incomplete, and I think it's maybe incomplete enough to be misleading. [Compare here](https://www.terraform.io/docs/enterprise/system-overview/data-security.html) — the object storage also has all the ingressed configuration versions and private modules, all of the binary plan files, and the run logs.

**Comment by ml4** [comment]
Agreed - thanks, I'd not seen the Data Security page.  I do however think it would be beneficial to link to the Data Security page to increase clarity around this.  Both a customer and, so it appears, me missed this.

File: content/source/docs/enterprise/admin/backup-restore.html.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ Backing up and restoring into a new installation of Terraform Enterprise is the
 
 ## About Backups
 
-Terraform Enterprise's backup utility backs up all of the data stored in a Terraform Enterprise installation, including both the blob storage and the PostgreSQL database. It does not back up the the installation configuration.
+Terraform Enterprise's backup utility backs up all of the data stored in a Terraform Enterprise installation, including both the blob storage (Terraform workspace state) and the PostgreSQL database. It does not back up the the installation configuration.
```
Comment (nfagerlund) [inline content/source/docs/enterprise/admin/backup-restore.html.md]: Okay, how about this: 

```suggestion
Terraform Enterprise's backup utility backs up all of the data stored in a Terraform Enterprise installation, including both the blob storage and the PostgreSQL database. It does not back up the the installation configuration.

See [Data Security](../system-overview/data-security.html) for details about the contents of Terraform Enterprise's blob storage and database.
```

**Comment by ml4** [comment]
👍

**Pull request merged**